integreat_cms/cms/models/abstract_content_translation.py

```diff
@@ -632,7 +633,7 @@ def __str__(self) -> str:
 
         :return: A readable string representation of the content translation
         """
-        return self.title
+        return str(self.title)
```

Comment:
```suggestion
        return self.title
```

self.title is already a string

Reply:
I received an error from pylint without the conversion. I was confused by this too, but didn't see a way around it